rules/unit-test-your-database/rule.md

```diff
@@ -23,12 +21,12 @@ With Visual Studio, you can write database unit tests. These are useful for test
 * User-defined functions
 * Views
 
-
 These tests can also be added to the same library as your unit, web and load tests.
 
-
 <!--endintro-->
 
-![Figure 1 - Database Unit Test](AddNewTest.jpg)  
+![Figure 1 - Database Unit Test](AddNewTest.jpg)
 
 ![Figure 2 - Writing the unit test against a stored proc](WriteUnitTest.jpg)
+
+If you want to know how to setup database unit tests locally and in your build pipeline, check out this article <https://cuteprogramming.wordpress.com/2022/04/29/unit-test-stored-procedures-and-automate-build-deploy-test-azure-sql-database-changes-with-ci-cd-pipelines/>
```

Comment:
Hey @AntPolkanov 

On your added link...

1. Can you follow https://ssw.com.au/rules/rule/#4-links
2. And this https://ssw.com.au/rules/descriptive-links/

Reply:
@tiagov8 fixed